### DISCUSSION TITLE
Stable swarm SVD ISSUE, Works in forge, why not swai.

### DISCUSSION THREAD

**Discussion by reponum8** [Stability-AI/StableSwarmUI#441] [Q&A]
Heres the issue, disregard clone, as i switched that on, the token issue starts straight away once svd 

![Screenshot (111)](https://github.com/user-attachments/assets/69732067-83fd-4f92-8b4f-ebcf21fcc230)

00:13:03.612 [Warning] ComfyUI-0 on port 7821 stderr:
00:13:04.246 [Info] User local requested 1 image with model 'OfficialStableDiffusion/svd_xt_1_1.safetensors'...
00:13:04.259 [Warning] ComfyUI-0 on port 7821 stderr: Traceback (most recent call last):
00:13:04.260 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 152, in recursive_execute
00:13:04.260 [Warning] ComfyUI-0 on port 7821 stderr:     output_data, output_ui = get_output_data(obj, input_data_all)
00:13:04.263 [Warning] ComfyUI-0 on port 7821 stderr:                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:04.263 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 82, in get_output_data
00:13:04.264 [Warning] ComfyUI-0 on port 7821 stderr:     return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
00:13:04.265 [Warning] ComfyUI-0 on port 7821 stderr:                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:04.265 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 75, in map_node_over_list
00:13:04.266 [Warning] ComfyUI-0 on port 7821 stderr:     results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
00:13:04.267 [Warning] ComfyUI-0 on port 7821 stderr:                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:04.268 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\nodes.py", line 575, in set_last_layer
00:13:04.268 [Warning] ComfyUI-0 on port 7821 stderr:     clip = clip.clone()
00:13:04.269 [Warning] ComfyUI-0 on port 7821 stderr:            ^^^^^^^^^^
00:13:04.270 [Warning] ComfyUI-0 on port 7821 stderr: AttributeError: 'NoneType' object has no attribute 'clone'
00:13:04.272 [Warning] ComfyUI-0 on port 7821 stderr:
00:13:08.609 [Info] User local requested 1 image with model 'OfficialStableDiffusion/svd_xt_1_1.safetensors'...
00:13:08.622 [Warning] ComfyUI-0 on port 7821 stderr: Traceback (most recent call last):
00:13:08.624 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 152, in recursive_execute
00:13:08.625 [Warning] ComfyUI-0 on port 7821 stderr:     output_data, output_ui = get_output_data(obj, input_data_all)
00:13:08.626 [Warning] ComfyUI-0 on port 7821 stderr:                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:08.626 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 82, in get_output_data
00:13:08.627 [Warning] ComfyUI-0 on port 7821 stderr:     return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
00:13:08.628 [Warning] ComfyUI-0 on port 7821 stderr:                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:08.628 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 75, in map_node_over_list
00:13:08.629 [Warning] ComfyUI-0 on port 7821 stderr:     results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
00:13:08.630 [Warning] ComfyUI-0 on port 7821 stderr:                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:08.630 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\nodes.py", line 575, in set_last_layer
00:13:08.631 [Warning] ComfyUI-0 on port 7821 stderr:     clip = clip.clone()
00:13:08.632 [Warning] ComfyUI-0 on port 7821 stderr:            ^^^^^^^^^^
00:13:08.632 [Warning] ComfyUI-0 on port 7821 stderr: AttributeError: 'NoneType' object has no attribute 'clone'
00:13:08.633 [Warning] ComfyUI-0 on port 7821 stderr:
00:13:31.998 [Info] User local requested 1 image with model 'OfficialStableDiffusion/svd_xt_1_1.safetensors'...
00:13:32.011 [Warning] ComfyUI-0 on port 7821 stderr: Traceback (most recent call last):
00:13:32.013 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 152, in recursive_execute
00:13:32.014 [Warning] ComfyUI-0 on port 7821 stderr:     output_data, output_ui = get_output_data(obj, input_data_all)
00:13:32.015 [Warning] ComfyUI-0 on port 7821 stderr:                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:32.015 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 82, in get_output_data
00:13:32.016 [Warning] ComfyUI-0 on port 7821 stderr:     return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
00:13:32.017 [Warning] ComfyUI-0 on port 7821 stderr:                     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:32.020 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\execution.py", line 75, in map_node_over_list
00:13:32.020 [Warning] ComfyUI-0 on port 7821 stderr:     results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
00:13:32.021 [Warning] ComfyUI-0 on port 7821 stderr:                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
00:13:32.022 [Warning] ComfyUI-0 on port 7821 stderr:   File "Z:\Aistudio_pipeline\StableSwarmUI\dlbackend\comfy\ComfyUI\nodes.py", line 575, in set_last_layer
00:13:32.022 [Warning] ComfyUI-0 on port 7821 stderr:     clip = clip.clone()
00:13:32.023 [Warning] ComfyUI-0 on port 7821 stderr:            ^^^^^^^^^^
00:13:32.024 [Warning] ComfyUI-0 on port 7821 stderr: AttributeError: 'NoneType' object has no attribute 'clone'

**Comment by mcmonkey4eva** [comment]
Did you miss the readme migration notice? https://github.com/Stability-AI/StableSwarmUI?tab=readme-ov-file#migration-notice
https://github.com/mcmonkeyprojects/SwarmUI/discussions/2
